src/vnsw/agent/test/test_util.cc

```diff
@@ -1633,6 +1636,7 @@ bool Inet4TunnelRouteAdd(const Peer *peer, const string &vm_vrf, char *vm_addr,
     Inet4TunnelRouteAdd(peer, vm_vrf, Ip4Address::from_string(vm_addr, ec), plen,
                         Ip4Address::from_string(server_ip, ec), bmap, label,
                         dest_vn_name, sg, path_preference);
+    return true;
```

Comment:
I think we should return the result of called function here:
```
return Inet4TunnelRouteAdd(peer, vm_vrf, Ip4Address::from_string(vm_addr, ec), plen,
                          Ip4Address::from_string(server_ip, ec), bmap, label,		                          Ip4Address::from_string(server_ip, ec), bmap, label,
                          dest_vn_name, sg, path_preference);		                          dest_vn_name, sg, path_preference);
```

Reply:
Same goes for other functions like this in this file.

Reply:
I don't think so. They always return true.

Reply:
This is an assumption which is true, but it is unnecessary. Someone could add a `return false;` path in the future after all. This shouldn't have passed the original CR, and just because it worked for all these years doesn't mean it's a code smell.